src/lib.rs

```diff
@@ -176,20 +188,30 @@ impl<C: Config> Default for GgrsPlugin<C> {
 
 impl<C: Config> Plugin for GgrsPlugin<C> {
     fn build(&self, app: &mut App) {
-        let mut schedule = Schedule::default();
-        schedule.set_build_settings(ScheduleBuildSettings {
-            ambiguity_detection: LogLevel::Error,
-            ..default()
-        });
-
         app.init_resource::<RollbackFrameCount>()
             .init_resource::<ConfirmedFrameCount>()
             .init_resource::<MaxPredictionWindow>()
             .init_resource::<RollbackOrdered>()
             .init_resource::<LocalPlayers>()
             .init_resource::<FixedTimestepData>()
-            .add_schedule(GgrsSchedule, schedule)
-            .add_schedule(ReadInputs, Schedule::new())
+            .init_schedule(ReadInputs)
+            .init_schedule(LoadWorld)
+            .init_schedule(SaveWorld)
+            .add_schedule(AdvanceWorld, {
+                let mut schedule = Schedule::default();
+                // AdvanceWorld is mostly a facilitator for GgrsSchedule, so SingleThreaded avoids overhead
+                // This can be overridden if desired.
+                schedule.set_executor_kind(ExecutorKind::SingleThreaded);
+                schedule
+            })
```

Comment:
Possibly scope creep: but we probably want to enable ambiguity detection (error) for these as well?